src/nlsolve_powerflow.jl

```diff
@@ -225,22 +198,24 @@ function _newton_powerflow(
     time_step::Int64;
     kwargs...)
 
-    # Fetch maxIterations, tol, and xtol from kwargs, or use defaults if not provided
+    # Fetch maxIterations, tol from kwargs, or use defaults if not provided
     maxIterations = get(kwargs, :maxIterations, DEFAULT_NR_MAX_ITER)
     tol = get(kwargs, :tol, DEFAULT_NR_TOL)
-    xtol = get(kwargs, :xtol, DEFAULT_NR_XTOL)  # default should be 0.0
```

Comment:
Why are we eliminating `xtol` altogether? This is a breaking change, right?

Reply:
I saw it causing wrong results